Nos/Views/StoriesView.swift

```diff
@@ -7,84 +7,84 @@
 
 import SwiftUI
 
+struct StoriesDestination: Hashable {
+    var author: Author?
+}
+
 struct StoriesView: View {
     
     @ObservedObject var user: Author
+
+    @EnvironmentObject var router: Router
     
     @FetchRequest private var authors: FetchedResults<Author>
     
-    @State private var currentAuthorIndex: Int = 0
-    @State private var scrollViewProxy: ScrollViewProxy?
+    @State private var selectedAuthor: Author
```

Comment:
Refactor that switches from storing the index to storing the actual collection element